lib/support/cli.js

```diff
@@ -240,6 +240,24 @@ module.exports.parseCommandLine = function parseCommandLine() {
       type: 'boolean',
       group: 'firefox'
     })
+    .option('firefox.geckoProfiler', {
```

Comment:
There's a pattern `feature` and `featureParams`:
```
    302:    .option('video', {
    307:    .option('videoParams.framerate', {
    312:    .option('videoParams.crf', {
    318:    .option('videoParams.addTimer', {
    324:    .option('videoParams.debug', {
    331:    .option('videoParams.keepOriginalVideo', {
    338:    .option('videoParams.filmstripFullSize', {
    345:    .option('videoParams.filmstripQuality', {
    350:    .option('videoParams.createFilmstrip', {
    356:    .option('videoParams.nice', {
    396:    .option('screenshot', {
    402:    .option('screenshotParams.type', {
    408:    .option('screenshotParams.png.compressionLevel', {
    413:    .option('screenshotParams.jpg.quality', {
    418:    .option('screenshotParams.maxSize', {
```
Let's follow that.

Reply:
Done